src/components/EmployeesList/EmployeesList.tsx

```diff
@@ -1,34 +1,30 @@
 import { Button, Table } from 'antd'
 import { ColumnsType } from 'antd/lib/table'
+import { ExpandableConfig } from 'antd/lib/table/interface'
 import React, { useState } from 'react'
 import { useLocation, useNavigate } from 'react-router-dom'
 import { ExpandedRow, UpdateButton } from './EmployeesList.style'
 import { useColumns } from './hooks/useColumns'
 
-export type Employee = {
-  key: string
-  firstName: string
-  lastName: string
-  email: string
-  department: string
-  specialization: string
-}
-
 type EmployeesListProps = {
+  isAdmin: boolean
```

Comment:
you should keep your user in some kind of store, my vote goes to context but you can choose whatever you like

Reply:
Yeah, they are kept in redux store. That's what I use redux for in this project